### PR TITLE
1249 update readme translations

### LINKED ISSUE #1249: Update 2.0 README translations
The Readme is our comprehensive sinatra documentation and deserves a lot of love. Thank you for everybody who has contributed to it so far!

The translations are great but some of them seem incomplete or outdated. For sinatra 2.0 they need a major overhaul and here's a list of languages that need to be worked on:

- [x] German
- [x] Spanish
- [ ] French
- [ ] Hungarian
- [x] Portuguese (Brasil/Portugal)
- [ ] Korean
- [x] Russian
- [x] Chinese

### PR DESCRIPTION
**Pull request by jeduardo824**
fix #1249 

PT-BR translation update. I made some corrections and additions. If anything is wrong, please, let me know. 
Can you do a quick review @Klockner?

### CODE REVIEW

**Comment by halan** [comment]
@jeduardo824 can you squash those commits with same message?

**Comment by jeduardo824** [comment]
Ok! I’ll do this tomorrow! Thanks for the feedback

File: README.pt-br.md [hunk 1]
```diff
@@ -33,18 +33,32 @@ Em seguida execute:
...
+
+<<<<<<< HEAD
+Códigos alterados só terão efeito após você reiniciar o servidor.  
+Por favor, reinicie o servidor após qualquer mudança ou use 
+[sinatra/reloader](http://www.sinatrarb.com/contrib/reloader).
+
+=======
+>>>>>>> b6f0256c... readme translation updates
+É recomendado também executar `gem install thin`. Caso esta gem esteja 
+disponível, o Sinatra irá utilizá-la.
```
Comment (namusyaka) [inline README.pt-br.md]: I think you're something wrong.. Please do the correct fix for conflicting.

Comment (jeduardo824) [reply README.pt-br.md]: Can you check if everything is alright now?

Comment (jeduardo824) [reply README.pt-br.md]: I'll try to fix this now.

**Comment by burningTyger** [comment]
I commented in #1249 There are still issues with the markup. Please fix and then :shipit:

**Comment by namusyaka** [comment]
@jeduardo824 Could you check @burningTyger's comment out?

**Comment by jeduardo824** [comment]
Just fixed it! Please, let me know if you detect something that stills wrong :) @burningTyger @namusyaka

**Comment by burningTyger** [comment]
There's still something wrong with it: https://github.com/sinatra/sinatra/blob/c68c80178f5bc9440cb5f50aaf0aa22481855fa2/README.pt-br.md#configura%C3%A7%C3%B5es-dispon%C3%ADveis

**Comment by jeduardo824** [comment]
@burningTyger Sorry!! I really didn't see this problem when I was translating the docs. It was just a missing `</tt>` tag. Fixed!

**Comment by burningTyger** [comment]
This looks good to me, thank you @jeduardo824. Unless any native speaker issues another concern I'd say this is good to go before we hit more changes.

Also, this will be my last translation issue. I have moved on to other projects. Thank you for all the maintainers who have kept sinatra alive for all those years. It's been my default framework for everything.

**Comment by jeduardo824** [comment]
@burningTyger Awesome! Good luck! Thank you for your patience!!

**Pull request merged**